avaframe/com1DFA/com1DFACfg.ini

```diff
@@ -286,6 +286,25 @@ B = 4.13
 #+++++++++++++Voellmy friction model
 xsi = 4000.
 
+#++++++++++Wet snow++++++++++++
+# 1 to activate, 0 not activated
+wetSnow = 0
+# reference enthalpy in J/kg to compute the new mu = mu_0 x exp(-enthalpy/enthRef)
+enthRef = 20000
+# initial temperature in °C
+TIni = -10
+# initial temperature for entrainment in °C
+entTempRef = -10.0
+# J/kgK (ice)
+cpIce = 2050.0
+# ToDo: Not used
+# J/kgK (water)
+cpWtr = 4187.0
```

Comment:
Let's just remove this here. And keep the comment in com1DFA.py. If that is ever reused, we can add this parameter back in...

Reply:
unused parameters removed